libControls/ToolTip.cpp

```diff
@@ -102,6 +129,12 @@ void ToolTip::draw() const
 		auto& renderer = NAS2D::Utility<NAS2D::Renderer>::get();
 		renderer.drawBoxFilled(rect(), NAS2D::Color::DarkGray);
 		renderer.drawBox(rect(), NAS2D::Color::Black);
-		renderer.drawText(mFont, mFocusedControl->second, position() + PaddingSize);
+
+		auto linePosition = position() + PaddingSize;
+
+		forEachLine(mFocusedControl->second, [this, &renderer, &linePosition](const std::string& lineStr) {
+			renderer.drawText(mFont, lineStr, linePosition);
+			linePosition.y += mFont.height();
+		});
```

Comment:
It's also possible to do a named capture, as an alternative to capturing `this`, and everything that has access to:
```cpp
[&font=mFont, &renderer, &linePosition]
```